### PR TITLE
Handle situation when coverimg fails to upload also added hints for uploading coverimage

### LINKED ISSUE #4139: Cover Image Upload: Handle Errors; Add Hints
Updates needed to Improve UX for Facility Cover Image Upload

- [ ] Add a hint in the modal, specifying the max. file size, allowed formats along with the recommended aspect ratio
- [ ] Handle Errors when uploading fails. Currently it shows loading, infinitely

### PR DESCRIPTION
**Pull request by patelaryan7751**
## Proposed Changes

- Fixes #4139 ?
- Added a hint stating the file size and format
image
image
- Handled the situation when uploading fails instead of infinite loading
image

@coronasafe/care-fe-code-reviewers @coronasafe/code-reviewers

## Merge Checklist

- [ ] Add specs that demonstrate bug / test a new feature.
- [ ] Update [product documentation](https://docs.coronasafe.network/coronasafe-care-documentation/architecture/architecture-and-layering-of-care).
- [ ] Ensure that UI text is kept in I18n files.
- [ ] Prep screenshot or demo video for changelog entry, and attach it to issue.
- [ ] Request for Peer Reviews
- [ ] Completion of QA

### CODE REVIEW

**Comment by patelaryan7751** [comment]
image

**Comment by nihal467** [comment]
LGTM

File: src/Components/Facility/CoverImageEditModal.tsx [hunk 1]
```diff
@@ -115,13 +128,22 @@ const CoverImageEditModal = ({
...
+                />
+              </div>
+              <p className="text-gray-700 font-medium text-center">
+                Max size for image uploaded should be 1mb.
+                <br />
+                Allowed formats are jpg,png,jpeg.
+                <br />
+                Recommended aspect ratio for facility cover photo is 1:1.
+              </p>
+            </>
```
Comment (khavinshankar) [inline src/Components/Facility/CoverImageEditModal.tsx]: this part seems to be repeating, why not refactor it

**Comment by nihal467** [comment]
@patelaryan7751 what is the status on the PR

File: src/Components/Facility/CoverImageEditModal.tsx (hunk 1 above)
Comment (patelaryan7751) [reply src/Components/Facility/CoverImageEditModal.tsx]: yeah refactored

**Review by khavinshankar — APPROVED** [review]

**Pull request merged**